### PR TITLE
Rebrand soroban-rpc to stellar-rpc

### PR DESCRIPTION
**Pull request by psheth9**
^^ title

### CODE REVIEW

File: docs/data/rpc/admin-guide.mdx [hunk 1]
```diff
@@ -224,7 +224,7 @@ By default, this is disabled (`sorobanRpc.persistence.enabled=false`) and the RP
 --set sorobanRpc.resources.limits.memory=2560Mi
```
Comment (jeesunikim) [inline docs/data/rpc/admin-guide.mdx:224]: are we going to change this command line soon?

File: docs/data/rpc/admin-guide.mdx [hunk 2]
```diff
@@ -305,15 +305,15 @@ docker run -p 8001:8001 -p 8000:8000 \
 
 #### Configuration
 
-For production, we recommend running Soroban RPC with a [TOML](https://toml.io/en/) configuration file rather than CLI flags. This is similar to creating a configuration file for Stellar-Core as we did previously. For example, using [our docker image](https://hub.docker.com/r/stellar/soroban-rpc):
+For production, we recommend running Stellar RPC with a [TOML](https://toml.io/en/) configuration file rather than CLI flags. This is similar to creating a configuration file for Stellar-Core as we did previously. For example, using [our docker image](https://hub.docker.com/r/stellar/soroban-rpc):
 
 ```bash
 docker run -p 8001:8001 -p 8000:8000 \
 -v <CONFIG_FOLDER>:/config stellar/soroban-rpc \
```
Comment (jeesunikim) [inline docs/data/rpc/admin-guide.mdx:312]: curious if we want to keep this as `soroban-rpc`

File: docs/data/rpc/admin-guide.mdx [hunk 3]
```diff
@@ -586,22 +586,22 @@ Make sure you configure the system test correctly:
...
 ```bash
 curl localhost:8001/metrics
 ```
 
 You will see many of the default Go and Process metrics (prefixed by `go_` and `process_` respectively) such as memory usage, CPU usage, number of threads, etc.
 
-We also expose metrics related to Soroban RPC (prefixed by `soroban_rpc`). There are many, but some notable ones are:
+We also expose metrics related to Stellar RPC (prefixed by `soroban_rpc`). There are many, but some notable ones are:
 
 - `soroban_rpc_transactions_count` - count of transactions ingested with a sliding window of 10m
```
Comment (jeesunikim) [inline docs/data/rpc/admin-guide.mdx:599]: same

**Comment by janewang** [comment]
@psheth9 Is this PR going in with P22 release?

File: docs/data/rpc/README.mdx [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: RPC Introduction
 sidebar_position: 5
 ---
 
-The Soroban-RPC can be described as a "live network gateway for Soroban." It provides information that the network currently has in its view (i.e. current state). It also has the ability to send a transaction to the network, and query the network for the status of previously sent transactions. It is meant to be simple, minimal, scalable, and familiar to blockchain developers from other ecosystems.
+The Stellar-RPC can be described as a "live network gateway for Soroban." It provides information that the network currently has in its view (i.e. current state). It also has the ability to send a transaction to the network, and query the network for the status of previously sent transactions. It is meant to be simple, minimal, scalable, and familiar to blockchain developers from other ecosystems.
 
 Soroban-RPC should provide all the basic data that a dapp developer would need, provided they are:
```
Comment (aditya1702) [inline docs/data/rpc/README.mdx:8]: Forgot to change it here and in the below bullet points

File: docs/data/rpc/admin-guide.mdx [hunk 5]
```diff
@@ -22,12 +22,12 @@ We don't recommend running the Quickstart image in production. See the [deploy y
...
 
 - [Stellar Core] - Node software that runs the network, coordinates consensus, and finalizes ledgers.
-- [Soroban RPC] server - JSON RPC server for interacting with Soroban contracts.
+- [Stellar RPC] server - JSON RPC server for interacting with Soroban contracts.
 - [Horizon server] - HTTP API for accessing ledger state and historical transactions.
 - [Friendbot server] - HTTP API for creating and funding new accounts on test networks.
 
 [Stellar Core]: https://github.com/stellar/stellar-core
-[Soroban RPC]: https://github.com/stellar/soroban-rpc/tree/main/cmd/soroban-rpc
+[Stellar RPC]: https://github.com/stellar/soroban-rpc/tree/main/cmd/soroban-rpc
```
Comment (aditya1702) [inline docs/data/rpc/admin-guide.mdx:30]: Are we not changing the URL too?

File: docs/data/rpc/admin-guide.mdx [hunk 6]
```diff
@@ -305,15 +305,15 @@ docker run -p 8001:8001 -p 8000:8000 \
...
 docker run -p 8001:8001 -p 8000:8000 \
 -v <CONFIG_FOLDER>:/config stellar/soroban-rpc \
 --config-path <RPC_CONFIG_FILENAME>
 ```
 
-You can use Soroban RPC itself to generate a starter configuration file:
+You can use Stellar RPC itself to generate a starter configuration file:
 
 ```bash
 docker run stellar/soroban-rpc gen-config-file > soroban-rpc-config.toml
```
Comment (aditya1702) [inline docs/data/rpc/admin-guide.mdx:319]: Same for here. Might want to change "stellar/soroban-rpc" to "stellar/stellar-rpc" and the .toml file naming too

**Review by aditya1702 — COMMENTED** [review]
Left a few comments in places where it is still being referenced by Soroban-rpc

**Comment by 2opremio** [comment]
I discussed it with @aditya1702  and we are closing it in favor of https://github.com/stellar/stellar-docs/pull/1064

**Pull request closed without merging**